README.md

```diff
@@ -270,6 +270,28 @@ To shut down the development server, use
 ./develop_server stop
 ```
 
+## Heroku Review Apps ##
+
+Heroku is a deployment service we use to run test versions of our app. Heroku
+can make a unique URL for each open pull request, where you can view a
+temporary version of the website with the changes from that pull request.
+
+### Permissions to access Heroku
+
+Access to Heroku isn't granted to everyone, but at least someone currently
+involved in the WiCS committee will ideally have the permissions. You don't
+need to be on the committee to have access. To gain access to Heroku, learn
+git, contribute regularly to the WiCS website (both opening pull requests and
+reviewing them), and then ask someone on WiCS syscom for access (either on
+GitHub or by mailing ```wics-sys@lists.uwaterloo.ca```)
+
+### To deploy a pull request
+
+1. Go to [https://dashboard.heroku.com/apps](https://dashboard.heroku.com/apps)
+2. Click on ```wics-site```
+3. In the left column, named ```Review Apps```, find the PR you want to deploy
+   and click the button that says ```Create Review App```
```

Comment:
I like the code formatting to show it as a clear label (I see it as a way of quoting something clearly) but maybe I'm misusing it. Also weird, the markdown editor I was using showed triple backticks same as single, but I'll update that :)